python/pyarrow/orc.py

```diff
@@ -197,6 +198,13 @@ def write_table(where, table):
     table : pyarrow.lib.Table
         The table to be written into the ORC file
```

Comment:
Can you reorder the docstring as well? :-)

Reply:
Ah, yes ;)